Version change to 0.11.1

ustulation opened the pull request:
Fix bug related to seed not getting printed if there was already a test that passed and then later another test failed.

afck commented on src/seeded_rng.rs:
```diff
@@ -107,7 +105,6 @@ impl SeededRng {
                 *seed_offset += 1;
                 Rc::new(RefCell::new(SeededRng {
                     seed: seed,
-                    print_seed_on_drop: false,
```
But this instance actually shouldn't print its seed, even if it happens to be the first one to be dropped: We only ever want the seed from line 54.

afck replied:
I think we actually just should set `seed` to the global `SEED` instead of the one that we pass into `XorShiftRng::from_seed`, then it should be fine.

ustulation committed:
Version change to 0.11.1
Fix bug related to seed not getting printed if there was already a test that passed and then later another test failed.
Add test to make sure toggling is correct w.r.t multiple panics.
